src/dodal/beamlines/i10.py

```diff
@@ -255,3 +257,48 @@ def idd_la_angle(
         wait=wait_for_connection,
         fake=fake_with_ophyd_sim,
     )
+
+
+@device_factory()
+def m1() -> Mirror_with_piezo:
+    return Mirror_with_piezo(prefix="BL10I-OP-COL-01:")
+
+
+@device_factory()
+def m3m5() -> Mirror_with_piezo:
+    return Mirror_with_piezo(prefix="BL10I-OP-SWTCH-01:")
+
+
+@device_factory()
+def m4() -> Mirror_with_piezo:
+    return Mirror_with_piezo(prefix="BL10I-OP-FOCS-01:")
+
+
+@device_factory()
+def s1() -> I10PrimarySlit:
```

Comment:
On i22 we've opted for slits_N, i03/i04 have s4_slit_gaps. I think it would be preferable if we had a consistent style: this is sleek with minimal typing, but it does also make it a little harder to see what the device. If the beamline are insistent I will fold, but I really do think slits_1 is worth the effort.

See also mirror_1